pages/challenges.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import Layout from '../components/Layout'
+import LessonTitleCard from '../components/LessonTitleCard'
+
+const setupAlert = (
```

Comment:
`setupAlert` is not a good name for 2 reasons:

1. `verb`. Components should be nouns, putting a verb here is misleading
2. Components should be Capitalized.

A better name: `Alert`